requirements/test.txt

```diff
@@ -7,11 +7,11 @@
 # Flake8 requirements
 flake8==2.3.0
 pyflakes==0.8.1
-pep8==1.5.7
+pep8==1.6.2
 mccabe==0.3
 
 fudge==1.0.3
 mock==1.0.1
-psutil==0.2.0
+psutil==2.2.1
```

Comment:
We should document that, but it's a pretty big task...

This particular package does not seem to be used anywhere. I ran a reverse dependency check and could not find a package requiring it. It was added in fe6836bc2ec68cc09fc5c2da63fca3596e13b047 (thanks, `git bisect`!) for qunit tests, which we no longer do. I'll remove it!